core/services/chainlink/config.go

```diff
@@ -67,6 +67,23 @@ func (c *Config) SetDefaults() {
 	//TODO terra and solana defaults https://app.shortcut.com/chainlinklabs/story/37975/chains-nodes-should-be-read-from-the-config-interface
 }
 
+type Secrets struct {
+	config.Secrets
+}
+
+func (s *Secrets) TOMLString() (string, error) {
+	b, err := toml.Marshal(s)
+	if err != nil {
+		return "", err
+	}
+	// remove runs of line breaks
+	str := multiLineBreak.ReplaceAllLiteralString(string(b), "\n")
+	// restore them preceding keys
+	str = strings.Replace(str, "\n[", "\n\n[", -1)
+	str = strings.TrimPrefix(str, "\n")
```

Comment:
What does this do?

Reply:
Pretty printing to strip repeated line breaks, but let's share a helper func instead of copying code?
https://github.com/smartcontractkit/chainlink/blob/develop/core/services/chainlink/config.go#L41-L53